diskplorer.py

```diff
@@ -29,21 +29,24 @@
                      help='Set SIZE as I/O buffer size for test (ex. 4k, 1M)', metavar='SIZE')
 optparser.add_option('-m', '--max-concurrency', dest='maxdepth', default=128, type='int',
                      help='Test maximum concurrency level N', metavar='N')
-optparser.add_option('-o', '--output', dest='output_filename', default='disk-concurrency-response.svg',
+optparser.add_option('-o', '--output', dest='output_filename', default='disk-concurrency-response-{buffer_size}-{test_name}.svg',
                      help='Write output graph to FILE', metavar='FILE')
 optparser.add_option('--raw-results', dest='raw_results_filename',
-                     metavar='FILE', default='disk-concurrency-response.csv',
+                     metavar='FILE', default='disk-concurrency-response-{buffer_size}-{test_name}.csv',
                      help='Write raw results (.csv) to FILE')
+optparser.add_option('-f','--file', dest='fio_json',
+                     metavar='FILE',
+                     help='Input file with fio results for processing')
 
 (options, args) = optparser.parse_args()
 
 mountpoint = options.mountpoint
 filesize = options.filesize
 maxdepth = options.maxdepth
 buffer_size = options.buffer_size
-output_filename = options.output_filename
-raw_filename = options.raw_results_filename
 test_name = options.test_name
+output_filename = options.output_filename.format(buffer_size=buffer_size,test_name=test_name)
```

Comment:
How would this work if I provided something like: `-o vladz_test`.

What I had in mind that string given with `-o` is then used for all output files: <name>.tmp, <name>.json, <name>.csv, <name>.svg.

This in particular make the `--raw-results` option not needed anymore.